examples/excelformer.py

```diff
@@ -1,20 +1,25 @@
-"""Reported (reproduced) accuracy(rmse for regression task) of ExcelFormer
+"""Reported (reproduced) accuracy (for multi-classification task), auc
+(for binary classification task) and rmse (for regression task)
 based on Table 1 of the paper https://arxiv.org/abs/2301.02819.
 ExcelFormer uses the same train-validation-test split as the Yandex paper.
-
-california_housing: 0.4587 (0.4733) num_layers=5, num_heads=4, num_layers=5,
-channels=32, lr: 0.001,
-jannis : 72.51 (72.38) num_heads=32, lr: 0.0001
-covtype: 97.17 (95.37)
-helena: 38.20 (36.80)
-higgs_small: 80.75 (65.17) lr: 0.0001
+The reproduced results are based on Z-score Normalization, and the
+reported ones are based on :class:`QuantileTransformer` preprocessing
+in the Sklearn Python package.
```

Comment:
What does this mean?

Reply:
It seems the data preprocessing here is Z-score Normalization (i.e., `(x - x.mean) / x.std`, I find it in `ExcelFormerEncoder` [line 684 of `pytorch-frame/torch_frame/nn/encoder/stype_encoder.py`](https://github.com/pyg-team/pytorch-frame/blob/master/torch_frame/nn/encoder/stype_encoder.py#L684)), it also exists before encoding of other baselines. Actually, the preprocessing in ExcelFormer (and also in Yandex's FT-Transformer) is [`quantile`](https://github.com/yandex-research/rtdl-revisiting-models/blob/main/output/adult/ft_transformer/default/0/stats.json#L7) implemented with `QuantileTransformer` in the Sklearn Python package. I don't know if I miss some key observations in this project, while these two preprocessing methods are indeed different and affect the results.

Since all baselines here uniformly use the Z-score Normalization in their encoders, thus I think the experimental results of Z-score Normalization is OK, and I comment the different preprocessing in the example script.

If it is necessary to change different preprocessing methods, opening a PR in the future is more suitable.

Reply:
I see! Can you clarify that these normalization is over numerical feature?

Reply:
Of course! I have clarified the normalization is over numerical feature in the example script.

The modification has been committed, if there is no further problem, I'd like to resolve all conversations to merge them.

I'd like to contribute [T2G-Former](https://github.com/jyansir/t2g-former) here in the next month, if there is no near arrangement for adding the `quantile` preprocessing, I can finish this part at the same time.

Reply:
Thanks. Feel free to merge!!

Reply:
All conversations have been resolved 🤝, thank you for the beneficial suggestions of package development. I have no write access, you can merge it.